compose/src/main/java/com/bottlerocketstudios/compose/appbar/ArchAppBar.kt

```diff
@@ -62,3 +58,9 @@ fun ArchAppBar(
         )
     }
 }
+
```

Comment:
Curious as to why this was moved below the composable function.  This makes it slightly harder to find.  Potentially much harder, if the file has a lot of composable functions after the main one.

Reply:
Discussed over standup. We will move the Different Data Class states and put it in its own file in the same package